packages/web/src/scss/components/Tooltip/README.md

```diff
@@ -0,0 +1,190 @@
+Add `tabindex="0"` to non-focusable elements to ensure keyboard accessibility.
+
+```html
+<div class="TooltipWrapper d-inline-block">
+  <span class="TooltipTarget" aria-describedby="my-tooltip-on-focus" tabindex="0">I have a tooltip!</span>
+  <div id="my-tooltip-on-focus" class="Tooltip Tooltip--top">
+    Hello there!
+    <span class="Tooltip__arrow"></span>
+  </div>
+</div>
+```
+
+Similar approach is also required for `disabled` elements. Disabled elements are
+not interactive which means users cannot hover, focus or click them to trigger a
+Tooltip. As a workaround, you'll want to trigger the Tooltip from a wrapper
+`<div>` or `<span>`, ideally made keyboard-focusable using `tabindex="0"`.
+
+```html
+<div class="TooltipWrapper d-inline-block">
+  <div class="TooltipTarget" aria-describedby="my-tooltip-for-disabled-button" tabindex="0">
+    <button type="button" disabled>I have a tooltip though I'm disabled</button>
+  </div>
+  <div id="my-tooltip-for-disabled-button" class="Tooltip Tooltip--top">
+    Hello there!
+    <span class="Tooltip__arrow"></span>
+  </div>
+</div>
+```
+
+### JavaScript
+
+It's not always possible to apply custom CSS class on an element, e.g. when
+using a component that doesn't accept custom class names. Or maybe you even want
+to trigger Tooltip programmatically from outside the target element. For this
+kind of task Tooltip responds to interaction classes `is-hidden` and
+`is-visible`.
+
+```html
+<button type="button" id="tooltip-trigger" data-target="#my-js-controlled-tooltip">Toggle tooltip</button>
+<div class="TooltipWrapper d-inline-block">
+  <div aria-describedby="my-js-controlled-tooltip">I have an externally-triggered tooltip</div>
+  <div id="my-js-controlled-tooltip" class="Tooltip Tooltip--top is-hidden">
+    Hello there!
+    <span class="Tooltip__arrow"></span>
+  </div>
+</div>
+```
+
+```js
+const toggleTooltip = (event) => {
+  const tooltipElement = document.querySelector(event.currentTarget.dataset.target);
+  tooltipElement.classList.toggle('is-hidden');
+  tooltipElement.classList.toggle('is-visible');
+};
+
+document.querySelector('#tooltip-trigger').addEventListener('click', toggleTooltip);
+```
+
+## Dismissible Tooltip [TODO JS plugin]
+
+Tooltip can be made dismissible by following these steps:
+
+1. Add `Tooltip--dismissible` modifier class on Tooltip.
+2. Add closing button with `Tooltip__close` class.
+3. Bind JS plugin [TODO] using `data-dismiss="tooltip"` and `data-target`
+   attributes on the closing button.
+
+```html
+<div id="my-dismissible-tooltip" class="Tooltip Tooltip--right Tooltip--dismissible">
+  Close me
+  <button
+    type="button"
+    class="Tooltip__close"
+    data-dismiss="tooltip"
+    data-target="#my-dismissible-tooltip"
+    aria-controls="my-dismissible-tooltip"
+    aria-expanded="true"
+  >
+    <span class="accessibility-hidden">Close</span>
+  </button>
+  <span class="Tooltip__arrow"></span>
+</div>
+```
+
+## Advanced Positioning
```

Comment:
@literat I suppose this should **not** be covered by the upcoming `dismiss` plugin (#DS-251), at least for the time being:

- it requires an external dependency,
- we can add it anytime later if necessary.